todoList.test.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * @jest-environment jsdom
+ */
+import todoList from './src/todoList.js';
+import { ToDo } from './src/ToDo.js';
+
+describe('Add Item', () => {
+  it('Add item to the DOM', () => {
+    document.body.innerHTML = '<div> <ul id="to-do"> </div>';
+    todoList.addTask(new ToDo('Finish the project', 0));
+    const list = document.querySelectorAll('#to-do li');
+    expect(list).toHaveLength(1);
+  });
+
+  it('Add item to the local Storage', () => {
```

Comment:
Please use `mock` for this function which is used to add items to localStorage.

Reply:
Fixed this problem.